scripts/claude-slide/generate_slide.py

```
@@ -36,6 +39,32 @@
 SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
 PROJECT_ROOT = os.path.normpath(os.path.join(SCRIPT_DIR, "..", ".."))
 
+
+def _find_native_cli_path() -> str | None:
+    """PATH上の claude、無ければ VS Code拡張が同梱するネイティブバイナリを探す。
+
+    claude_agent_sdk はまず自分のパッケージに同梱された claude CLI を使おうとするが、
+    そのバイナリは x86_64 でビルドされている。Apple Silicon 上では Rosetta 経由になり、
+    Bunランタイムが「CPU lacks AVX support, strange crashes may occur」という警告を出し
+    実際に生成が無言のまま止まる（ハングする）事象を確認している。
```

Ruff (0.16.3)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)